rules/gcp_audit_rules/gcp_iam_service_accounts_get_access_token_privilege_escalation.py

```diff
@@ -0,0 +1,20 @@
+from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_walk
```

Comment:
deep_walk is a builtin function of event, so you can do `event.deep_walk("protoPayload"...` without importing from panther_base_helpers

Reply:
I'm confused
if we use https://github.com/panther-labs/panther-analysis/blob/release/rules/gcp_audit_rules/gcp_privilege_escalation_by_deployments_create.py#L16 as a reference - it imports `deep_walk`